Formula/clipsafe.rb

```diff
@@ -3,6 +3,7 @@ class Clipsafe < Formula
   homepage "https://waxandwane.org/clipsafe.html"
   url "https://waxandwane.org/download/clipsafe-1.1.tar.gz"
   sha256 "7a70b4f467094693a58814a42d272e98387916588c6337963fa7258bda7a3e48"
+  license "GPL-2.0-only"
```

Comment:
I think it is less restrictive GPL-2.0-or-later. 
The download is the single `clipsafe` script with license
```
# cliPSafe is a command line interface to Password Safe databases.
# Copyright (C) 2008  Ross Palmer Mohn, rpmohn@waxandwane.org
#
# This program is free software; you can redistribute it and/or
# modify it under the terms of the GNU General Public License
# as published by the Free Software Foundation; either version 2
# of the License, or (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
```